source/nodejs/adaptivecards/src/card-elements.ts

```diff
@@ -3345,11 +3345,11 @@ export class NumberInput extends Input {
         this._numberInputElement = document.createElement("input");
         this._numberInputElement.setAttribute("type", "number");
 
-        if (this.min) {
```

Comment:
Feature request for typescript: this should compile down to a null/undefined check. Truthy/falsy values only cause pain, with very little gain.

Reply:
So true